bigflow/build/operate.py

```diff
@@ -0,0 +1,147 @@
+"""Actual implementaion of buid/distribution operations"""
+
+import os
+import subprocess
+import shutil
+import typing
+import logging
+import typing
+
+from pathlib import Path
+
+import bigflow.resources
+import bigflow.dagbuilder
+import bigflow.version
+import bigflow.build.pip
+import bigflow.build.dev
+import bigflow.build.dist
+import bigflow.build.dataflow.dependency_checker
+import bigflow.commons as bf_commons
+
+from bigflow.build.spec import BigflowProjectSpec
+
+
+logger = logging.getLogger(__name__)
+
+
+def run_tests(prj: BigflowProjectSpec):
+    logger.info('Runing tests...')
+    output_dir = "build/junit-reports"
+    try:
+        bf_commons.run_process([
+            "python", "-m", "xmlrunner", "discover",
+            "-s", ".",
+            # "-t", project_dir,
```

Comment:
To remove?

Reply:
Maybe. For now I keep it.
Eventually I want to replace `python -m xmlrunner` with `pytest`.